apps/console/src/features/server-configurations/constants/server-configurations-constants.ts

```diff
@@ -321,6 +329,10 @@ export class ServerConfigurationsConstants {
 	public static readonly OFFLINE_PASSWORD_RESET: string = "Recovery.AdminPasswordReset.Offline";
 	public static readonly ADMIN_FORCED_PASSWORD_RESET_EXPIRY_TIME: string = "Recovery.AdminPasswordReset.ExpiryTime";
 
+    //multi attribute login set
```

Comment:
Shall we format this comment properly? Start with an uppercase letter and ends with a full stop.